src/material-examples/tsconfig-build.json

```diff
@@ -24,8 +24,10 @@
     "paths": {
       "@angular/material/*": ["../../dist/packages/material/*"],
       "@angular/material": ["../../dist/packages/material/public-api"],
+      "@angular/material-experimental/*": ["../../dist/packages/material-experimental/*"],
```

Comment:
We should actually just move the tree into src/lib now that we're on for next major